native/SampleApps/RestAPIExplorer/RestAPIExplorer/Classes/AppDelegate.m

```diff
@@ -115,7 +120,9 @@ - (void)initializeAppViewState
 - (void)setupRootViewController
 {
     RestAPIExplorerViewController *rootVC = [[RestAPIExplorerViewController alloc] initWithNibName:nil bundle:nil];
-    self.window.rootViewController = rootVC;
+    UINavigationController *navViewController = [[UINavigationController alloc] initWithRootViewController:rootVC];
+    [[SFAppStyler sharedInstance] styleNavigationBar:navViewController.navigationBar];
+    self.window.rootViewController = navViewController;
```

Comment:
This is pretty wonky.  It creates an artificial nav bar above an existing header bar in the root view controller.  The styling should be applied to the existing `UIToolBar` in the root view controller, if at all.  The nav controller configured here should be removed.